static/src/javascripts/projects/common/modules/video/events.js

```diff
@@ -119,25 +121,6 @@ define([
         return action;
     }
 
-    function buildGoogleAnalyticsEvent(mediaEvent, metrics, canonicalUrl) {
```

Comment:
but its a helper module 😉. As far as I can tell its only used there too (albeit passed in as an argument).